compiler_tester/src/vm/eravm/lambda_vm_adapter.rs

```diff
@@ -98,12 +100,15 @@ pub fn run_vm(
 
         lambda_contract_storage.insert(key, bytecode_u256);
     }
+    let initial_storage = InitialStorageMemory {
+        initial_storage: lambda_storage,
+    };
+    let storage_before_run = initial_storage.clone();
```

Comment:
Is this variable really needed? Can't we just keep `initial_storage` and `.clone()` it when using for `EraVM::new()`?

Reply:
Done